suplemon/viewer.py

```diff
@@ -96,13 +97,22 @@ def get_size(self):
         y, x = self.window.getmaxyx()
         return (x, y)
 
+    def get_scroll_pos(self):
+        return (self.y_scroll, self.x_scroll)
+
+    def get_y_scroll(self):
+        return self.y_scroll
+
+    def get_x_scroll(self):
```

Comment:
Mainly the same idea as in `get_config_dir`. I would like to encourage using methods to access stuff so I can change the implementation details easier. What do you think?